### ISSUE TITLE
Naming convention Microsoft Entra

### ISSUE THREAD

**Issue by FLX-0x00** [RIPAGlobal/omniauth-azure-activedirectory-v2#21]
Is there any effort to rename the gem because microsoft has officially renamed the azure active directory to Entra: https://learn.microsoft.com/en-us/entra/fundamentals/new-name

Have a great day!

**Comment by pond** [comment]
It's a good point - but no, definitely not, as this would break everything that uses the gem, just for a whims of a marketing department who might change their mind again in a year anyway.

I fully recognise that it's awkward, especially if you don't know the history of the Microsoft names. Suggestions about how we might make this gem more discoverable by people looking for "entra" are welcome, but renaming would IMHO just be too disruptive.

**Comment by FLX-0x00** [comment]
Totally agree. Spent nealy 5 hours researching to find the right gem (office365, ms365, entra, azure ad, openid, saml, oauth2, whatever) so many names and microsoft still not updated their own documentation - was horror for me to implement SSO for that. I did some OpenID Connect stuff with monkey patches that worked, and then I found this gem, but I was not sure if it worked with MS Entra. So I overlooked that this is maintained and works very well! So maybe add some hint to the readme and help other people who are searching like me.

Something like: We know Microsoft is switching to Entra as the new name for their product, but it uses the same endpoints and protocols, it will work!
